src/helpers.c

```diff
@@ -461,6 +461,50 @@ void remove_trailing_slash(char *url)
 	}
 }
 
+static bool adjust_system_time()
+{
+	int err;
+	time_t clear_time, system_time;
+	char *time_str;
+
+	err = verify_time(globals.path_prefix, &system_time, &clear_time);
+	if (err == -ENOENT) {
+		/* in the case we are doing an installation to an empty directory
+	/	 * using swupd verify --install, we won't have a valid versionstamp
+		 * in path_prefix, so try searching without the path_prefix */
+		err = verify_time(NULL, &system_time, &clear_time);
+	}
+
+	if (err == 0) {
+		// System time is correct
+		return true;
+	}
+
+	if (err == -ENOENT) {
+		error("Failed to read system timestamp file\n");
+	}
+
+	time_str = time_to_string(system_time);
+	if (time_str) {
+		printf("Current time is %s\nAttempting to fix...\n", time_str);
```

Comment:
I find this message a little confusing:
```
Current time is Mon May  4 09:39:22 PDT 2020
Attempting to fix...
```
First, in this sentence "Current time is X", is that the time that is incorrect that is going to be fixed? or is that the correct time that the wrong time is going to be corrected to?

Secondly, I think "attempting to fix..." should be "attempting to fix it...".

Reply:
Updated to:
```Fixing outdated system time from 'Tue May 05 09:03:42 2020' to 'Fri May 01 11:47:27 2020'```